### PR TITLE
logo added

### PR DESCRIPTION
**Pull request by mikey-roberts**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,14 +3,17 @@ import React from "react";
...
-
 require("dotenv").config();
 
 export default function App() {
 
   return (
     <div className ='app'>
-      <h1>Pin My Hike</h1> 
+      <div className="title-overlay">
+                <img className="PMH-logo"
```
Comment (YinnyF) [inline client/src/App.js:13]: indentation 😱

File: client/src/components/maps/MyMap.js [hunk 2]
```diff
@@ -3,21 +3,27 @@ import "../../styles/App.css";
...
-  height: '80vh'
+  width: '100vw', 
+  height: '100vh'
 }
 const center = {
   lat: 55.378052,
   lng: -3.435973
 }
 
+console.log(process.env.REACT_APP_GOOGLE_MAPS_API_KEY)
```
Comment (YinnyF) [inline client/src/components/maps/MyMap.js:22]: can remove this

File: client/src/App.js (hunk 1 above)
Comment (mikey-roberts) [reply client/src/App.js:13]: Yus, will no fix immediately. Thoughts of PR has brought about complacency.. 1000 apologies

**Review by YinnyF — APPROVED** [review]
👍

**Pull request merged**